README.md

```diff
@@ -520,6 +520,63 @@ This last conditional block is simply to ignore the unneeded license files, base
 
 > :warning: Note that `include` and `exclude` are still mutually exclusive even if they are in different, but included, conditional sections.
 
+## Useful for template authors
+
+> Available since version [0.9.0]
+
+As a template author you're probably concerned about successful builds of your template?
+
+Imagine a couple of month after your first template release, some new versions of any dependencies would break your template, and you would not even be aware of it?
```

Comment:
month => months... plural :-)

Reply:
now you know who is also a very comprehensive writer..